hypervisor/arch/x86/cpu_caps.c

```diff
@@ -394,7 +396,7 @@ static int32_t check_vmx_mmu_cap(void)
  * we should supplement which feature/capability we must support
  * here later.
  */
-int32_t hardware_detect_support(void)
+int32_t check_hardware_support(void)
```

Comment:
check usually means operation only. It doesn't mean if the check result is success/failure...
detect means if something exist.
use detect_hardware_support is better

Reply:
ok, will update.